aiida/manage/configuration/profile.py

```diff
@@ -140,72 +112,36 @@ def uuid(self):
             return None
 
     @uuid.setter
-    def uuid(self, value):
+    def uuid(self, value: Optional[str]):
         self._attributes[self.KEY_UUID] = value
 
     @property
-    def default_user(self):
-        return self._attributes.get(self.KEY_DEFAULT_USER, None)
+    def default_user_email(self) -> Optional[str]:
+        return self._attributes.get(self.KEY_DEFAULT_USER_EMAIL, None)
 
-    @default_user.setter
-    def default_user(self, value):
-        self._attributes[self.KEY_DEFAULT_USER] = value
+    @default_user_email.setter
+    def default_user_email(self, value):
```

Comment:
Should this setter than also allow `None`?

Reply:
not sure what you mean by allow None?
I just added a type to the `value`, to specify it can be `str|None`,
other than that there is nothing stopping it being set to None.

Reply:
That was my question though, whether we should allow it being set to `None`. But since that is the current behavior let's leave that as it is for now.